R/cache.R

```diff
@@ -252,12 +252,12 @@ cache_epidata_call <- function(epidata_call, fetch_args = fetch_args_list()) {
     if (as_of_recent || issues_recent) {
       cli::cli_warn(
         c(
-          "using cached results with `as_of` within the past week (or the future!).",
-          "This will likely result in an invalid cache. Consider\n",
-          "1. disabling the cache for this session with `disable_cache` or permanently with environmental ",
-          "variable `EPIDATR_USE_CACHE=FALSE`\n",
-          "2. setting `EPIDATR_CACHE_MAX_AGE_DAYS={Sys.getenv('EPIDATR_CACHE_MAX_AGE_DAYS', unset = 1)}`",
-          " to e.g. `3/24` (3 hours)."
+          "Using cached results with `as_of` within the past week (or the future!).
+          This will likely result in an invalid cache. Consider",
+          "i" = "disabling the cache for this session with `disable_cache` or
+          permanently with environmental variable `EPIDATR_USE_CACHE=FALSE`",
+          "i" = "setting `EPIDATR_CACHE_MAX_AGE_DAYS={Sys.getenv('EPIDATR_CACHE_MAX_AGE_DAYS
+          ', unset = 1)}` to e.g. `3/24` (3 hours)."
```

Comment:
todo: This rendered as `ℹ setting `EPIDATR_CACHE_MAX_AGE_DAYS=1` to e.g. `3/24` (3 hours).` for me.  Tweak to remove or reword the `=1`.

Reply:
I think that was intended, to show the current value? though it does look like github is treating the ` strangely.

Reply:
maybe we reword it for clarity?

```
setting `EPIDATR_CACHE_MAX_AGE_DAYS` (currently {Sys.getenv('EPIDATR_CACHE_MAX_AGE_DAYS'
, unset = 1)}) to e.g. `3/24` (3 hours)
```

Reply:
it might also benefit from some extra explanation what it's doing, making sure that they receive any new versions of versions quickly (but with better wording)